api/v1alpha1/serviceresolver_types_test.go

```diff
@@ -103,248 +125,35 @@ func TestServiceResolver_MatchesConsulTrue(t *testing.T) {
 					},
 				},
 				ConnectTimeout: 1 * time.Second,
-			},
-		},
-	}
-	for name, c := range cases {
-		t.Run(name, func(t *testing.T) {
-			require.True(t, c.Ours.MatchesConsul(c.Theirs))
-		})
-	}
-}
-
-// Test MatchesConsul for cases that should return false.
-func TestServiceResolver_MatchesConsulFalse(t *testing.T) {
```

Comment:
Curious, why are you removing this test?

Reply:
I removed it because it was testing `reflect.DeepEqual`. Although the "different type" case is still valid to test I think.

Reply:
@ashwin-venkatesh should we add these back in https://github.com/hashicorp/consul-k8s/pull/340?

Reply:
can do